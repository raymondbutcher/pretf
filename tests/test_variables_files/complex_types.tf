variable "aws_credentials" {
  type = map({
    nonprod = map(string)
    prod    = map(string)
  })
  default = {
    nonprod = {
      profile = "pretf-nonprod"
    }

    prod = {
      profile = "pretf-prod"
    }
  }
}
