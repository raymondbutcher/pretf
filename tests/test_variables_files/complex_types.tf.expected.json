[
    {
        "name": "aws_credentials",
        "default": {
            "nonprod": {
                "profile": "pretf-nonprod"
            },
            "prod": {
                "profile": "pretf-prod"
            }
        },
        "source": "complex_types.tf"
    }
]